scripts/vendor-boringssl.sh

```diff
@@ -188,7 +188,7 @@ PATTERNS=(
 'crypto/*/*/*.S'
 'crypto/*/*/*/*.c'
 'third_party/fiat/*.h'
-'third_party/fiat/*.c'
+#'third_party/fiat/*.c'
```

Comment:
is this commented rather than deleted deliberately?

Reply:
Yes: the presence of C files in this directory changes over time, and I got sick of deleting and re-adding this line all the time, so I just commented it out.